swagger.yaml

```diff
@@ -2500,6 +2500,105 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /clusters/{cluster_id}/operators:
+    get:
+      tags:
+        - operators
+        - installer
+      security:
+        - agentAuth: []
```

Comment:
From the security role i understand that this API will be used by the agent? 
If so isn't it expected to be returned as part of cluster get?

Reply:
That's correct. It is controller-service API. Why is it expected to be returned by cluster GET?
Some logic happens in the BE. The cluster returns the` operator-status` list.